source/stylesheets/refills/unstyled/_accordion-tabs.scss

```diff
@@ -0,0 +1,36 @@
+.accordion-tabs {
+  $large-screen: em(860) !default;
+
+  @include clearfix;
+  padding: 0;
+
+  .tab-header-and-content {
+    list-style: none;
+
+    @include media($large-screen) {
+      display: inline;
+    }
+  }
+
+  .tab-link {
+    display: block;
+    padding-right: 1em;
+
+    @include media($large-screen) {
+      display: inline-block;
+    }
+  }
+
+  .is-active {
+    color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.